preprocessing/README.md

```diff
@@ -0,0 +1,36 @@
+# Preprocessing
+
+This folder contains raw market data as published by cities and scripts to
+convert it into GeoJSON format.
+
+
+## Usage example (Berlin)
+
+1. Download the latest raw market data and put in into the *raw*
+folder as *markets-berlin.json*.
+2. Convert the raw market data by running the following command from
+the project root folder:
+
+	``` bash
+	$ preprocessing/berlin/compile-berlin-geojson.js
```

Comment:
Please update `package.json` with the necessary dev dependencies (`moment`, `opening_hours`).